rmw_fastrtps_cpp/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ find_package(ament_cmake REQUIRED)
 find_package(fastrtps_cmake_module REQUIRED)
 find_package(fastcdr REQUIRED CONFIG)
 find_package(fastrtps REQUIRED CONFIG)
+find_package(FastRTPS REQUIRED MODULE)
```

Comment:
@dirk-thomas yeah, I completely agree, I think this hodgepodge of CMake modules is a bad smell, that's why I asked:

> What would you suggest? Improve FindFastRTPS.cmake and get rid of the CONFIG lines or just keep this as is?

I'll see if I can get rid of the the `find_package(... CONFIG)` lines and do everything in the `FindFastRTPS` module.